future-apps/agent/test/agent.js

```diff
@@ -407,18 +425,18 @@ contract('Agent app', (accounts) => {
         await setDesignatedSignerContract(true, false, false, false)
 
         // Signature fails check
-        assertIsValidSignature(false, await agent.isValidSignature(HASH, NO_SIG))
+        assertIsValidSignature(false, await agent.isValidSignature(HASH, ERC1271_SIG))
       })
 
-      it('isValidSignature returns false if designated signer doesnt support the interface', async () => {
+      it('isValidSignature returns true even if the designated signer doesnt support the interface', async () => {
         // false - not ERC165 interface compliant
         // true  - any sigs are valid
         // false - doesn't revert on checking sig
         // false - doesn't modify state on checking sig
         await setDesignatedSignerContract(false, true, false, false)
 
         // Requires ERC165 compliance before checking isValidSignature
```

Comment:
Should amend this comment too; from what I've understood EIP 1271 doesn't require EIP 165?